app/addons/fauxton/assets/less/components.less

```diff
@@ -44,6 +40,7 @@
 
 }
 
-a.clipboard-copy-element:hover {
-  text-decoration: none;
-}
+button.clipboard-copy-element {
+  background: transparent;
+  border: 0;
+}
```

Comment:
trailing whitespace at EOF missing

you can set your editor to automatically add them